rollingpin/main.py

```diff
@@ -5,6 +5,7 @@
 from twisted.internet.defer import inlineCallbacks, returnValue
 from twisted.internet.task import react
 
+from rollingpin.elasticsearch import enable_elastic_search_notifications
```

Comment:
nit: please use the explicit relative imports like the rest of this code and put this import in the right place alphabetically